"undefined: syscall.SyncFileRange" when compiling for armv7 processors

Trying to build a program I'm writing that uses pebble to run on my Raspberry Pi 3B+ gives me the following error:

```
# github.com/petermattis/pebble/vfs
../../../../go/pkg/mod/github.com/petermattis/pebble@v0.0.0-20190702160538-d011cead4df8/vfs/syncing_file_linux.go:39:33: undefined: syscall.SyncFileRange
../../../../go/pkg/mod/github.com/petermattis/pebble@v0.0.0-20190702160538-d011cead4df8/vfs/syncing_file_linux.go:76:9: undefined: syscall.SyncFileRange
```

--- a/internal/arenaskl/node.go
+++ b/internal/arenaskl/node.go
@@ -65,11 +65,11 @@ func newNode(
 		panic("height cannot be less than one or greater than the max height")
 	}
 	keySize := key.Size()
-	if keySize > math.MaxUint32 {
+	if int64(keySize) > math.MaxUint32 {
 		panic("key is too large")
 	}
 	valueSize := len(value)
-	if len(value) > math.MaxUint32 {
+	if int64(len(value)) > math.MaxUint32 {
 		panic("value is too large")
 	}
 

--- a/vfs/syncing_file_generic.go
+++ b/vfs/syncing_file_generic.go
@@ -2,7 +2,7 @@
 // of this source code is governed by a BSD-style license that can be found in
 // the LICENSE file.
 
-// +build !linux
+// +build !linux arm
 
 package vfs
 

--- a/vfs/syncing_file_linux.go
+++ b/vfs/syncing_file_linux.go
@@ -2,7 +2,7 @@
 // of this source code is governed by a BSD-style license that can be found in
 // the LICENSE file.
 
-// +build linux
+// +build linux,!arm
 
 package vfs
 

--- a/vfs/syncing_file_linux_test.go
+++ b/vfs/syncing_file_linux_test.go
@@ -2,7 +2,7 @@
 // of this source code is governed by a BSD-style license that can be found in
 // the LICENSE file.
 
-// +build linux
+// +build linux,!arm
 
 package vfs
 

